components/layout/GetStartedStep.tsx

```diff
@@ -22,19 +22,21 @@ export default function GetStartedStep({
   ...props
 }: GetStartedStepProps) {
   return (
-    <Paper sx={{ textAlign: 'center', p: 4 }} {...props}>
+    <Paper sx={{ textAlign: 'left', p: 4 }} {...props}>
       <Typography variant="h2" sx={{ mb: 4 }}>
         {title}
       </Typography>
       {bodyText && <MuiMarkdown>{bodyText}</MuiMarkdown>}
-      <SectionContainer sx={{ textAlign: 'left', my: 3 }}>
+      <SectionContainer sx={{ textAlign: 'left', my: 3, px: 0 }}>
         {children}
       </SectionContainer>
 
       {ctaText && ctaLink && (
-        <Button href={ctaLink} variant="contained" sx={{ my: 3 }}>
-          {ctaText}
-        </Button>
+        <Stack direction="row" justifyContent="center">
```

Comment:
This code is almost identical to Ln 33-37 in components/layout/PaperSection.tsx. It might be worth creating a reusable component

Reply:
That is a good eye @sramalho94 but I am just wondering if this is something we might use again, or if it might just be another component to maintain. I see where you are going with this but since this is pretty small maybe not so worth it? Maybe for the Button with the CTA text and link though, I can see where that might be used a lot more places.

Reply:
Thank you for your great suggestions and catching the off margins.
I applied all the comments except this one. This one can be a different task.
I think the PR is ready for review and merging.

Reply:
I have used rebase instead of merging and I think i have resolved conflicts when I was rebasing (E.g. you can find aria labels that are added to the staging exist in the new commits)
I think with rebasing, we can only use it with force push.

Can I still apply new commits? It seems like this pull request is being merged.
Please let me know if there are merge conflicts.

Reply:
Hey all! When we use the `rebase` we have to resolve conflicts ( as @rezamshy seems to have resolved) and to push we need to `force`.
Thats why those forces appears.

But I just didnt understand why so many force pushes :thinking: 
![image](https://github.com/clearviction-devs/clearviction-wa/assets/80538553/7e812f20-1f4f-4b11-a66a-ac52c9969b4a)

All those force pushes were because staging was updated with some new code and  you had to rebase again @rezamshy ?

Reply:
I'm not sure what they are for either. I might have used the last command for pushing in my bash. But the reason for them is not that staging has changed during that time.